api/pkg/cli/cmd/search/search.go

```diff
@@ -121,6 +122,11 @@ func (opts *options) run() error {
 }
 
 func (opts *options) validate() error {
+
+	if len(opts.Args) == 0 && len(opts.Kinds) == 0 && len(opts.Tags) == 0 {
```

Comment:
With this check will it return an error to this as well `tkn hub search <task-name>`

Reply:
nope